src/server/plugins/register-form/index.js

```diff
@@ -17,18 +17,22 @@ exports.register = function(server, options, next) {
   const stateConfig = cookies.encryptedCookies(!server.settings.app.debug);
   stateConfig.options.path = server.realm.modifiers.route.prefix || '/';
 
+  const routeConfig = _.merge({
+    plugins: { crumb: true }
```

Comment:
could just hardcode this into the `encryptedCookies` config but I like the idea of keeping concerns about csrf and cookies seperate